ha/k8s_setup/ha_setup.py

```diff
@@ -188,6 +188,10 @@ def process(self):
         Process config command.
         """
         try:
+            # Get log path from cluster.conf.
+            log_path = Conf.get(self._index, f'cortx{_DELIM}common{_DELIM}storage{_DELIM}log')
+            ha_log_path = os.path.join(log_path, f'ha/{Conf.machine_id}')
```

Comment:
does  machine_id here mean the machine id for HA pod ? 

If yes,   is it available in Conf directly like this ? if yes, @Madhura-08  do we need to fetch it again on line 263  ?

Reply:
>> does machine_id here mean the machine id for HA pod ?
Yes, 
For creating log path as **<confstore-log-path>/<component>/<machine-id>/.log** files.
here I am using Confstore api 'Conf.machine_id'  to get machine_id for that specific pod.

```
[root@ssc-vm-g3-rhev4-2114 cortx-ha-1]# python3
Python 3.6.8 (default, Nov 16 2020, 16:55:22)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from cortx.utils.conf_store import Conf
>>>
>>> Conf.machine_id
'bbb340f79047df9bb52fa460615c32a5'
>>>

```

Reply:
okay. @shaileshsvc  @Madhura-08 in this case we can remove the get_machine_id function altogether right ?

Reply:
Instead of fetching machine_id at two diff places, 
Defined machine_id at line 193, and using same at all over the process() function.

Reply:
I think yes.